test/helper.rb

```diff
@@ -71,13 +73,17 @@ def assert_parser_unchanged(asset)
       file = File.basename(asset, '.y')
 
       result = Dir.chdir(PROJECT_DIR) do
-        File.read("#{REGRESS_DIR}/#{file}.rb") == File.read("#{TAB_DIR}/#{file}")
+        d1 = File.read("#{REGRESS_DIR}/#{file}.rb")
+        d2 = File.read("#{TAB_DIR}/#{file}")
+        l1 = Ripper.lex(d1).map {|e| e[1,2] unless [:on_comment,:on_nl,:on_ignored_nl,:on_sp].include?(e[1]) }.compact
+        l2 = Ripper.lex(d2).map {|e| e[1,2] unless [:on_comment,:on_nl,:on_ignored_nl,:on_sp].include?(e[1]) }.compact
+        l1 == l2
       end
 
       assert(result, "Output of test/assets/#{asset} differed from " \
         "expectation. Try compiling it and diff with test/regress/#{file}.rb:" \
         "\nruby -I./lib ./bin/racc -o tmp/#{file} test/assets/#{asset}; " \
-        "colordiff tmp/#{file} test/regress/#{file}.rb")
+        "lex_diff tmp/#{file} test/regress/#{file}.rb")
```

Comment:
`colordiff` is an actual CLI tool. On Ubuntu, you can install it with `sudo apt-get install colordiff`. How about `lex_diff`?